admin/client/src/lib/CastStringToElement.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
```

Comment:
hmm yeah, Git doesn't want to detect renamed cases to files... I will need to rebase the initial commit to fix this